styles.css

```diff
@@ -0,0 +1,30 @@
+.main {
+  display: flex;
+}
+.search {
+  margin: 15px;
+  line-height: 30px;
+}
+.editor {
+  box-shadow: 1px 3px 5px grey;
+  line-height: 30px;
+  width: 40vw;
+  height: 50vh;
+  border: 1px solid black;
+  display: inline-block;
+  padding: 15px;
+  margin: 15px;
+}
```

Comment:
When you have the same properties in different classes you can avoid repetition (and keep your code DRY) using something like this:

```
.editor,
.page {
  border: 1px solid black;
  display: inline-block;
  padding: 15px;
  margin: 15px;
}
```